community/embedded-examples/src/docs/dev/logging.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[tutorials-java-embedded-logging]]
+= Controlling logging
+
+:javadoc-graphdatabasefactory: javadocs/org/neo4j/graphdb/factory/GraphDatabaseFactory.html
+:javadoc-slf4jlogprovider: javadocs/org/neo4j/logging/slf4j/Slf4jLogProvider.html
+:javadoc-neo4jlogging: javadocs/org/neo4j/logging
+
+Neo4j provides logging via its own +link:{javadoc-neo4jlogging}/Log.html[org.neo4j.logging.Log]+ layer, and does not natively use any existing Java logging framework.
```

Comment:
I'd probably clarify this to say that it's wrt to embedded Neo4j. It's unlikely we'd change it in the embedded case, but certainly in the server case we wouldn't guarantee that.